esmvaltool/cmorizers/obs/cmorize_obs_cds_uerra.py

```diff
@@ -0,0 +1,175 @@
+    # so explicitly get them here.
+    cube.coord('longitude').guess_bounds()
+
+    # Set correct names
+    cube.var_name = definition.short_name
+
+    cube.long_name = definition.long_name
+
+    # Convert units if required
+    cube.convert_units(definition.units)
+
+    # Set global attributes
+    utils.set_global_atts(cube, attributes)
+
+    logger.info("Saving CMORized cube for variable %s", cube.var_name)
+    utils.save_variable(cube, cube.var_name, out_dir, attributes)
+
+    return in_file
+
+
+def _regrid_dataset(in_dir, var, cfg):
+    """
+    Regridding of original files.
+
+    This function regrids each file and write to disk appending 'regrid'
+    in front of filename.
+    """
+    workdir = cfg['work_dir']
+    # Match any year here
+    filepattern = var['file'].format(year='????')
+    filelist = glob.glob(os.path.join(in_dir, filepattern))
+    for infile in filelist:
+        _, infile_tail = os.path.split(infile)
+        outfile = os.path.join(workdir, infile_tail)
+        targetgrid_ds = xr.DataArray.from_iris(
+            _stock_cube(cfg['custom']['regrid']))
+        input_ds = xr.open_dataset(infile)
+        # Do renaming for consistency of coordinate names
+        input_ds = input_ds.rename({'latitude': 'lat', 'longitude': 'lon'})
+        # Select uppermoist soil level (index 0)
+        input_da = input_ds[var['raw']].isel(soilLayer=0)
+        logger.info("Regridding... ")
+        # A workaround to avoid spreading of nan values,
+        # related to Github issue
+        constantval = 10
+        input_da = input_da + constantval
+        assert int((input_da == 0.).sum()) == 0  # Make sure that there
+        # are no zero's in the data,
+        # since they will be masked out
+        regridder = xe.Regridder(input_ds,
+                                 targetgrid_ds,
+                                 'bilinear',
+                                 reuse_weights=True)
+        da_out = regridder(input_da)
+        da_out = da_out.where(da_out != 0.)
+        da_out = da_out - constantval
+
+        # Save it.
+        logger.info("Saving: %s", outfile)
+        da_out.to_netcdf(outfile)
+
+
+def cmorization(in_dir, out_dir, cfg, cfg_user):
+    """Cmorization func call."""
+    # run the cmorization
+    # Pass on the workdir to the cfg dictionary
+    cfg['work_dir'] = cfg_user['work_dir']
+    # If it doesn't exist, create it
+    if not os.path.isdir(cfg['work_dir']):
+        logger.info(f"Creating working directory for regridding: {cfg['work_dir']}")
+        os.mkdir(cfg['work_dir'])
+
+    for short_name, var in cfg['variables'].items():
+        var['short_name'] = short_name
+        logger.info(f"Processing var {short_name}")
+
+        # Regridding
+        logger.info("Start regridding to: %s", cfg['custom']['regrid'])
+        _regrid_dataset(in_dir, var, cfg)
+        logger.info("Finished regridding")
+
+        logger.info("Start CMORizing")
+        for year in range(1979, 2029):
```

Comment:
```suggestion
        for year in range(1961, 2029):
```
The dataset starts in 1961.

Reply:
Ah cool, it has been expanded already.